arangod/Aql/QueryRegistry.h

```diff
@@ -109,6 +109,10 @@ class QueryRegistry {
    *        information.
    */
   void setNoLockHeaders(ExecutionEngine* engine) const;
+  
+ public:
+  /// @brief default timeout for holding read-locks on a collection
+  static constexpr double defaultReadLockTimeout = 900;
```

Comment:
shouldn't this be defined somewhere else ?

Reply:
Ok, where does it belong then? Any suggestions?

Reply:
I actually don't feel confident that this change was a good idea. I am reverting this part and setting the default timeout back to 0.0, which will use some default timeout (whatever this is).